src/Juno/Consensus/Commit.hs

```diff
@@ -67,7 +67,7 @@ applyCommand :: Monad m => UTCTime -> Command -> Raft m (NodeID, CommandResponse
 applyCommand tEnd cmd@Command{..} = do
   apply <- view (rs.applyLogEntry)
   logApplyLatency cmd
-  result <- apply _cmdEntry
+  result <- apply (_cmdEntry, _cmdRequestId)
```

Comment:
You're right, that is more future proof. I'll switch to passing the entire command in.